docs/main/user-guides/local-storage-user-guide/local-pv-zfs/faq.md

```diff
@@ -0,0 +1,293 @@
+---
```

Comment:
Are the below contents repeated in the master faq?

Reply:
no